cmd/ingress/ingress.go

```diff
@@ -39,6 +41,15 @@ func dief(template string, args ...interface{}) {
 	os.Exit(1)
 }
 
+func waitForSignal(stopCh chan struct{}) {
+	sigCh := make(chan os.Signal, 1)
+	signal.Notify(sigCh, syscall.SIGINT, syscall.SIGTERM)
```

Comment:
We can add `SIGABRT` and `SIGKILL`, too

Reply:
Processes cannot prevent `SIGKILL`.

I think we shouldn't change the behavior of these two.

Reply:
May be `SIGKILL` only trigged  by people, but `SIGABRT` can be trigged by Kubernetes.

Reply:
In which case Kubernetes will send `SIGABRT`? As far as i know, it only sends `SIGTERM` and `SIGKILL`.

Reply:
It seems that I made a mistake. 
`SIGABRT` could be sent manually by people to view the stack trace.
`SIGKILL ` will be sent by Kubernetes when container is still running after the grace period.

Reply:
I think we still need to add `SIGABRT` and `SIGKILL `.

Reply:
Behavior of SIGKILL cannot be changed, it's kernel's limitation.

I cannot understand why we should capture SIGABRT only because people may send it, if so, wo have to capture all signals since people may send them as well.

Reply:
> Behavior of SIGKILL cannot be changed, it's kernel's limitation.

Capture signals for logging, `SIGKILL` would be sent by Kubernetes. why not to capture?

Reply:
OK，got it